modules/tensor_mechanics/src/materials/LAROMANCEStressUpdateBase.C

```diff
@@ -768,6 +768,27 @@ LAROMANCEStressUpdateBaseTempl<is_ad>::computeResidual(
   for (unsigned int p = 0; p < _num_partitions; p++)
   {
     if (_partition_weights[p])
+      {
+        // compute weight normalizing factor
+        GenericReal<is_ad> weight_normalizer = 0;
+        unsigned int number_of_active_tiles = 0;
+        for (unsigned int t = 0; t < _num_tiles[p]; ++t)
+        {
+          // tile normalization factor (sum of tile weights)
+          weight_normalizer += _weights[p][t];
+          // count number of active tiles
+          if (_weights[p][t] > 0)
```

Comment:
Also, this can be simplified to `if (_weights[p][t])` since it's equivalent. Up to you, just fyi